pkg/gatewayserver/io/basicstationlns/messages/upstream.go

```diff
@@ -175,11 +175,6 @@ func (req *JoinRequest) ToUplinkMessage(ids ttnpb.GatewayIdentifiers, bandID str
 		rxTime = &val
 	}
 
-	ids.EUI, err = GetEUIfromUID(ids.GatewayID)
-	if err != nil {
-		return nil, errGatewayID.WithAttributes("id", ids.GatewayID).WithCause(err)
```

Comment:
Not sure if we need this error still, but afaik we _always_ use unique IDs otherwise it doesn't say anything

Reply:
You're right, the error is unused. I did a search for `errGatewayID` and found some references in `pkg/gatewayserver/io/basicstationlns` but apparently that error is redefined in that package. Will remove.